src/test/java/com/solvd/micro9/users/integration/graphql/GraphqlUserControllerIT.java

```diff
@@ -0,0 +1,132 @@
+package com.solvd.micro9.users.integration.graphql;
+
+import com.solvd.micro9.users.TestUtils;
+import com.solvd.micro9.users.domain.aggregate.User;
+import com.solvd.micro9.users.domain.es.EsStatus;
+import com.solvd.micro9.users.domain.es.EsType;
+import com.solvd.micro9.users.domain.es.EsUser;
+import com.solvd.micro9.users.web.controller.GraphqlUserController;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.graphql.GraphQlTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.graphql.execution.ErrorType;
+import org.springframework.graphql.test.tester.GraphQlTester;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+@GraphQlTest(GraphqlUserController.class)
+@Import(GraphqlUserControllerITConfig.class)
+public class GraphqlUserControllerIT {
+
+    @Autowired
+    private GraphQlTester tester;
+
+    @Test
+    void verifyAllUsersAreFoundTest() {
+        String query = "{ getAllUsers { id firstName lastName email phone } }";
+        List<User> result = this.tester.document(query)
+                .execute()
+                .path("data.getAllUsers[*]")
+                .entityList(User.class)
+                .get();
+        List<User> users = List.of(TestUtils.getUser());
+        assertUsersList(users, result);
+    }
+
+    @Test
+    void verifyUsersAreFondByCriteriaTest() {
```

Comment:
@eyakauleva In JUnit 5, It's not necessary, to add suffix `Test` to the method's signature

Reply:
fixed 83582da